api/src/modules/import-data/sourcing-data/sourcing-data-import.service.ts

```diff
@@ -112,13 +114,21 @@ export class SourcingDataImportService {
           materials,
           dtoMatchedData.sourcingData,
         );
-
       // TODO: TBD What to do when there is some location where we cannot determine its admin-region: i.e coordinates
       //       in the middle of the sea
       const geoCodedSourcingData: SourcingData[] =
         await this.geoCodingService.geoCodeLocations(
           sourcingDataWithOrganizationalEntities,
         );
+      const warnings: string[] = [];
```

Comment:
Double check if import should be stopped when a location is not accurately geocoded (multiple results)

Reply:
Jus checked that the import should continue adding the geolocation warning both to the Task keeping track of the import process and also to the Sourcing Location entity